functions.php

```diff
@@ -56,4 +58,25 @@ function displayPokemon(array $pokemons): string {
         }
     }
     return $result;
-}
\ No newline at end of file
+}
+
+/**
+ * Adds data to the database
+ *
+ * @param none
+ *
+ * @return array, when returned the data will be sent to the database
+ */
+function addPokemonToDatabase()
+{
+    $dataBase = getDatabaseObject();
+    $pokedexNumber = $_POST['addPokedexNo'];
+    $pokemonName = $_POST['addPokemonName'];
+    $pokemonType = implode(' ', $_POST['addPokemonType']);
+    $pokemonImage = $_POST['addPokemonImage'];
+    $insert = $dataBase->prepare('INSERT INTO `kanto_pokedex` (`pokedex_no`, `name`, `type`, `img_source`) VALUES (?, ?, ?, ?);');
+    $insert->execute([$pokedexNumber, $pokemonName, $pokemonType, $pokemonImage]);
+    return $insert;
```

Comment:
why? what is this used for?

Reply:
i didn't have it originally but you said all functions end with a return so i just put it in to fit that defintion, should i change my function?

Reply:
rather than returning this, which u dont use, maybe think about what would be useful to return. the execute function will tell you if the query has worked or not with a boolean return value. grab that value and check if it was successful, then show the user a success or a failure message based on that

Reply:
updated